tests/step-016.sh

```diff
@@ -25,8 +27,8 @@ mkdir -p /tmp/test16 && cd /tmp/test16 || exit 1
 git init || exit 1
 
 mkdir -p .githooks &&
-    git config --global githooks.shared '/tmp/shared/hooks-016-a.git' &&
-    echo '/tmp/shared/hooks-016-b.git' >.githooks/.shared &&
+    git config --global githooks.shared 'file:///tmp/shared/hooks-016-a.git' &&
```

Comment:
The one on the git config should still work with the local path, right?
Can you please change those back so we get coverage on that code path as well?

Reply:
True, I can look over the test and change these back, the ones which should still work. Its also tests in 117.

Reply:
Done, its only 016 which makes sense